Changelog.md

```diff
@@ -11,6 +11,7 @@ Bugfixes:
  * Commandline interface: Do not try creating paths ``.`` and ``..``.
  * Type system: Fix a crash caused by continuing on fatal errors in the code.
  * Type system: Disallow arrays with negative length.
+ * Inline assembly: Charge one stack slot for non-value types during analysis.
```

Comment:
It is a bit sloppy, but value types indeed occupy two stack slots while non-value types occupy only one. The reason is that what is put on the stack here is not the value but the address of the variable (so that it can be combined with SLOAD). As value types do not always occupy a full slot, we have to supply the intra-slot offset.